WordPress/Sniff.php

```diff
@@ -109,64 +109,205 @@ abstract class Sniff implements PHPCS_Sniff {
+		'gzseek'                                 => true,
+		'gzwrite'                                => true,
+		'hash_update_stream'                     => true,
+		'http_support'                           => true,
+		'hypot'                                  => true,
+		'id3_get_genre_id'                       => true,
+		'id3_get_version'                        => true,
+		'idate'                                  => true,
+		'imagefontheight'                        => true,
+		'imagefontwidth'                         => true,
+		'imagegetinterpolation'                  => true,
+		'imagetypes'                             => true,
+		'intdiv'                                 => true,
+		'intl_get_error_code'                    => true,
+		'intlcal_field_difference'               => true,
+		'intlcal_get_actual_maximum'             => true,
+		'intlcal_get_actual_minimum'             => true,
+		'intlcal_get_day_of_week_type'           => true,
+		'intlcal_get_first_day_of_week'          => true,
+		'intlcal_get_greatest_minimum'           => true,
+		'intlcal_get_least_maximum'              => true,
+		'intlcal_get_minimal_days_in_first_week' => true,
+		'intlcal_get_minimum'                    => true,
+		'intlcal_get_now'                        => true,
+		'intlcal_get_repeated_wall_time_option'  => true,
+		'intlcal_get_skipped_wall_time_option'   => true,
+		'intlcal_get_time'                       => true,
+		'intlcal_get_weekend_transition'         => true,
+		'intlgregcal_get_gregorian_change'       => true,
+		'intltz_count_equivalent_ids'            => true,
+		'intltz_get_dst_savings'                 => true,
+		'intltz_get_error_code'                  => true,
+		'intltz_get_offset'                      => true,
+		'intltz_get_raw_offset'                  => true,
+		'json_last_error'                        => true,
+		'lcg_value'                              => true,
+		'levenshtein'                            => true,
+		'log'                                    => true,
+		'log10'                                  => true,
+		'log1p'                                  => true,
+		'mb_ereg_search_getpos'                  => true,
+		'mb_strwidth'                            => true,
+		'mb_substr_count'                        => true,
+		'mhash_count'                            => true,
+		'microtime'                              => true,
+		'mt_getrandmax'                          => true,
+		'mt_rand'                                => true,
+		'ob_get_level'                           => true,
+		'ord'                                    => true,
+		'pi'                                     => true,
+		'post_type_archive_title'                => true,
+		'print'                                  => true,
+		'printf'                                 => true,
+		'rand'                                   => true,
+		'random_int'                             => true,
+		'readonly'                               => true,
```

Comment:
This one is [deprecated](https://developer.wordpress.org/reference/functions/readonly/) and shouldn't be used.